tests/test_webhook_loop_prevention.py

```
@@ -0,0 +1,493 @@
+class TestMonitoringAndLogging:
+    """Test that loop prevention events are properly logged."""
+
+    @patch('src.mvp.webhook.monitoring')
+    def test_bot_detection_logged(self, mock_monitoring, client, mock_verify_signature):
+        """Test that bot detection is logged to monitoring."""
+        payload = create_webhook_payload(
+            user_login="poggio-bot[bot]",
+            user_id=2634514
+        )
+
+        response = client.post(
```

Ruff (0.14.14)

[error] Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)